### ISSUE TITLE
Legacy Prefix not being detected when random tags present

### ISSUE THREAD

**Issue by mpnewcomb** [allegro/axion-release-plugin#439]
The legacy 'release-' prefix is not being detected when there are random tags present. I had to specify:
```
scmVersion {
  tag {
    prefix = 'release'
    versionSeparator = '-'
  }
}
```
for it to pick up my existing legacy tags. I have another project that just has legacy tags and it picks them up fine.

**Comment by sfitts** [comment]
In our environment using `1.13.6` even specifying the tag explicitly did not work.  The only solution was to remove the non-conforming tags.

**Comment by bgalek** [comment]
Hi, I comfirm a bug, hope to fix it someday ;)

**Comment by sfitts** [comment]
As of `1.14.0` it has gotten worse.  Now even on a repo with no non-conforming tags I cannot get it to recognize the legacy format.  Specifying the tag prefix and separator does not work either.

**Comment by bgalek** [comment]
It should be an easy fix, to ignore not versions related tags, @sfitts wanna make PR? :)

**Comment by spyroid** [comment]
Yes, I need something similar: to skip incorrect tag. Now it's just failing when see 'release--' instead of 'release-'
